source/httpfs/faileddriver.go

```diff
@@ -0,0 +1,41 @@
+package httpfs
```

Comment:
This file should be removed once the other changes have been made

Reply:
This is still used by the `New()` function. Lets revisit this once we agree on general layout of this package.